libcontainer/cgroups/fs2/fs2.go

```diff
@@ -105,6 +105,10 @@ func (m *manager) GetStats() (*cgroups.Stats, error) {
 	if err := statMemory(m.dirPath, st); err != nil && !os.IsNotExist(err) {
 		errs = append(errs, err)
 	}
+	// memory event
+	if err := eventMemory(m.dirPath, st); err != nil && !os.IsNotExist(err) {
+		errs = append(errs, err)
+	}
```

Comment:
Can we have an integration test?
https://github.com/opencontainers/runc/tree/main/tests/integration